tasks/init.rb

```diff
@@ -0,0 +1,47 @@
+#!/opt/puppetlabs/puppet/bin/ruby
+#
+# Puppet Task to purge nodes
+# This can only be run against the Puppet Master.
+#
+# Parameters:
+#   * agent_certnames - A comma-separated list of agent certificate names.
+#
+require 'puppet'
+require 'open3'
+
+Puppet.initialize_settings
+
+unless Puppet[:server] == Puppet[:certname]
+  puts 'This task can only be run against the Master (of Masters)'
```

Comment:
I'm not sure this is true for this module. It can run against any node that has the puppetdb terminus package installed and configured:

https://github.com/acidprime/exports/blob/a3a4967ce823c86a0f108cddf1e472305c0732a4/lib/puppet/face/node/exports.rb#L44-L55

Reply:
Hm, ok, I'll take off this check,